utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/universal/RecordFormatterSupport.scala

```diff
@@ -22,7 +22,7 @@ class RecordFormatterSupportDispatcher(kafkaConfig: KafkaConfig, schemaRegistryC
 
 }
 
-trait RecordFormatterSupport {
+sealed trait RecordFormatterSupport {
```

Comment:
Why it has to be sealed now? Why is it necessary?

Reply:
I've use it in pattern matching along the way and thought since all extensions to that trait are located in this file we might as well make it sealed. What is more I don't think we need to extend that trait in any other place since we have two schema types supported right now and it is a good idea to keep those formatters in one place.

Reply:
no problem :)